frontend/www/js/omegaup/components/problem/creator/cases/MultipleCasesInput.vue

```diff
@@ -49,15 +58,23 @@ import T from '../../../../lang';
 export default class MultipleCasesInput extends Vue {
   multipleCasesPrefix = '';
   multipleCasesSuffix = '';
-  multipleCasesCount = 0;
+  multipleCasesCount = 1;
   multipleCasesGroup: GroupID = NIL;
 
   T = T;
 
   get caseNamePreview() {
-    return `${this.multipleCasesPrefix + '1' + this.multipleCasesSuffix}, ${
-      this.multipleCasesPrefix + '2' + this.multipleCasesSuffix
-    }, ...`;
+    return `${this.multipleCasesPrefix}1${this.multipleCasesSuffix}, ${this.multipleCasesPrefix}2${this.multipleCasesSuffix}...`;
+  }
+
+  // Ensure that the prefix and suffix always contain alpha-numeric characters in addition to _ and -
+  formatter(text: string) {
+    return text.toLowerCase().replace(/[^a-z0-9_\-\t]/g, '');
```

Comment:
```suggestion
    return text.toLowerCase().replace(/[^a-zA-Z0-9_-]/g, '');
```

sería mejor no incluir tabs (el `\t`), pero sí incluir mayúsculas (también, si el `-` va al final, no necesita escaparse!)